api/stream/stream.go

```diff
@@ -30,7 +29,7 @@ type API struct {
 // pong command.
 func New(pingPeriod, pongTimeout time.Duration, allowedWebSocketOrigins []string) *API {
 	return &API{
-		clients:     make(map[uint][]*client),
+		clients:     make(map[uint]map[string]*client),
```

Comment:
could more than one client be connected with the same token? Your implementation would only allow one to be addressable.